src/pages/documentation/index.md

```diff
@@ -7,63 +7,56 @@ contributors:
 
 # Mobile SDK documentation
 
-<InlineAlert variant="success" slots="header, text"/>
-
-Good to go for iOS 15
-
-Adobe Experience Platform Mobile SDKs have been functionally tested on iOS 15 release candidate versions. Please make sure you use [current SDK versions](#) to ensure compatibility.
-
-<InlineAlert variant="info" slots="header, text"/>
+Welcome to the home of Adobe Experience Platform Mobile SDK documentation.
 
-Are you interested in Kotlin?
-
-The Adobe Mobile SDK team is looking for feedback on better supporting customers that are currently or planning to use Kotlin in their mobile applications. If you're interested in a Kotlin-based version of the Adobe Experience Platform Mobile SDK, please fill out the [following survey](https://forms.office.com/r/5E5d7pJivG).
-
-<InlineAlert variant="info" slots="header, text"/>
+<InlineAlert variant="success" slots="header, text"/>
 
-The Swift versions of the iOS SDKs have been released! 
+Good to go for iOS 16 and Android 13
 
-For more information on the Swift versions, including available Swift extensions and reference information, please read the [current SDK versions document](#).<br/><br/>Additionally, the Swift iOS SDKs are open source. You can learn more about this move to Swift and open source in the [Adobe Tech Blog](https://medium.com/adobetech/adobe-experience-platform-mobile-sdks-move-to-swift-for-ios-6aa67b67b4d4).
+Latest versions of the Adobe Experience Platform SDK for Google Android and Apple iOS (Swift & Objective-C) are tested for compatibilit with release verisons of Apple iOS 16 and Google Android 13. See [Current SDK Versions](./current-sdk-versions.md) to find our latest SDK versions or [release notes](./release-notes/index.md) for information on changes and updates.
```

Comment:
A couple of misspelled words here - should be "The latest versions of the Adobe Experience Platform SDK for Google Android and Apple iOS (Swift and Objective-C) are tested for compatibility with release versions of..."

Reply:
updated.